src/test/java/org/openrewrite/kotlin/tree/KTSTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.kotlin.tree;
+
+import org.junit.jupiter.api.Test;
+import org.junitpioneer.jupiter.ExpectedToFail;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.kotlin.Assertions.kotlinScript;
+
+public class KTSTest implements RewriteTest {
+
+    @ExpectedToFail("KTS parser is not implemented yet")
```

Comment:
Why do we need this annotation?

Reply:
Because this test is to verify Kotlin compiler can parse the .kts correctly, but the entire test is expected to be failed since we don't support .kts parsing right now.